lib/migration.js

```diff
@@ -0,0 +1,493 @@
+/*
+ * Copyright (c) 2012-2016 Digital Bazaar, Inc. All rights reserved.
+ */
+
+var _ = require('lodash');
+var async = require('async');
+var bedrock = require('bedrock');
+var config = bedrock.config;
+var database = require('./database');
+var progress = require('progress');
+
+var BedrockError = bedrock.util.BedrockError;
+
+var logger = bedrock.loggers.get('app');
+
+var api = {};
+module.exports = api;
+
+// steps per progress bar tick call optimization
+var PROCESSED_PER_TICK = 20;
+
+bedrock.events.on('bedrock-cli.init', function() {
```

Comment:
This should move into `api.main()` too since otherwise every app using bedrock-mongodb will see these migration options.